src/api-docs/source/includes/_wallet_rpc_auctions.md

```diff
@@ -1334,6 +1341,998 @@ address | Required | address submitting the bid
+</aside>
+
+```shell--vars
+name='possibility'
+amount=5.000000
+lockup=10.000000
+```
+
+```shell--cli
+hsw-cli rpc createbid $name $amount $lockup
+```
+
+```javascript
+const {WalletClient} = require('hs-client');
+const {Network} = require('hsd');
+const network = Network.get('regtest');
+
+const clientOptions = {
+  port: network.walletPort,
+  apiKey: 'api-key'
+}
+
+const client = new WalletClient(clientOptions);
+
+(async () => {
+  const result = await client.execute('createbid', [ name, amount, lockup ]);
+  console.log(result);
+})();
+```
+> createbid returns JSON structured like this: 
+
+```json
+{
+  "hash": "85bce02cc5cb8ba9ff4e23d84ce389310f50074a6b9a8b20b8643a56a4cb9f9a",
+  "witnessHash": "78a33edde952c80dd05476afe1e5e6e0a942484feb914e8ecf4c4b66329940a9",
+  "mtime": 1537402434,
+  "version": 0,
+  "inputs": [
+    {
+      "prevout": {
+        "hash": "5f7892816226f3f753dfd003eea4f565fa699a5b29bafde27b6628b1a2966910",
+        "index": 0
+      },
+      "witness": [
+        "bfab12daf81378ad40c46d037a52ffd4f3374c6f71cd6997b067ea9b498e29ac359cf9b267265f31741b28916a7d3da3021ca60539473d59cef3dc88b25c9e9801",
+        "03b9843c1c40c210790e55052c3e8c56b49d2f0f1d00e8bdeb0237f076a128365a"
+      ],
+      "sequence": 4294967295,
+      "address": "rs1q9jd8fgq8xa0drggyd6azggpeslacdzx5gr04ss"
+    }
+  ],
+  "outputs": [
+    {
+      "value": 10000000,
+      "address": "rs1qj3jpnvrs8afqmhpqanvlyc7m25duaxj4txaru5",
+      "covenant": {
+        "type": 3,
+        "action": "BID",
+        "items": [
+          "01c05e8ea3d1c347342ef11c50fe5a1f621c942f7f8f7e0ee329eb883f93f9eb",
+          "d3070000",
+          "706f73736962696c697479",
+          "001361beb541240738829f101e842c76b9666d1bc4a87299a8ed6a9d2127ed61"
+        ]
+      }
+    },
+    {
+      "value": 990004940,
+      "address": "rs1qd7xx0qnn5qpnmcuadhnz99q6sydfgq8fvg6e4v",
+      "covenant": {
+        "type": 0,
+        "action": "NONE",
+        "items": []
+      }
+    }
+  ],
+  "locktime": 0,
+  "hex": "00000000015f7892816226f3f753dfd003eea4f565fa699a5b29bafde27b6628b1a296691000000000ffffffff0280969800000000000014946419b0703f520ddc20ecd9f263db551bce9a5503042001c05e8ea3d1c347342ef11c50fe5a1f621c942f7f8f7e0ee329eb883f93f9eb04d30700000b706f73736962696c69747920001361beb541240738829f101e842c76b9666d1bc4a87299a8ed6a9d2127ed61cc46023b0000000000146f8c678273a0033de39d6de622941a811a9400e90000000000000241bfab12daf81378ad40c46d037a52ffd4f3374c6f71cd6997b067ea9b498e29ac359cf9b267265f31741b28916a7d3da3021ca60539473d59cef3dc88b25c9e98012103b9843c1c40c210790e55052c3e8c56b49d2f0f1d00e8bdeb0237f076a128365a"
+}
+```
+Create `BID` transaction without signing or broadcasting it.
+
+### Params
+Name | Default |  Description
+--------- | --------- | ---------
+name | Required | name to bid on
+amount | Required | amount to bid (in HNS)
+lockup | Required | amount to lock up to blind your bid (must be greater than bid amount)
```

Comment:
in the code this is actually called `value` (the value of the actual TX output). Since our RPC uses positional parameters, the names from the client perspective don't matter. But if we ever actually comply with JSON 2.0 and allow _named parameters_ (looks more like a JSON object) then we will have to change this to `value` to be correct. What are your thoughts? Come back and address it later? I know amount/lockup makes way more sense to users at this time.

Reply:
Rename "value" in the implementation to "amount".

Reply:
gonna leave this one alone I think, it does make more sense to document it as "lockup" but we may need to revisit if we ever allow named arguments with rpc calls.